sequencing/ATACSeq/jobSubmission/2_batchCalcQCMetrics.sh

```diff
@@ -6,46 +6,68 @@
 #SBATCH --nodes=1 # specify number of nodes.
 #SBATCH --ntasks-per-node=16 # specify number of processors per node
 #SBATCH --mail-type=END # send email at job completion 
-#SBATCH --output=ATACSeq/logFiles/%u/calcATACQC-%A_%a.o
-#SBATCH --error=ATACSeq/logFiles/%u/calcATACQC-%A_%a.e
-#SBATCH --job-name=calcATACQC
+#SBATCH --output=ATACSeq/logFiles/%u/ATACcalcQCS2-%A_%a.log
+#SBATCH --error=ATACSeq/logFiles/%u/ATACcalcQCS2-%A_%a.err
```

Comment:
Could you remove %u from these so that it doesn't require the user to have created their own log files directory? The movement of log files later in the script accounts for the files not being in the correct spot yet. Considering the log files are going to be moved at the start of run time, these log files can be placed anywhere.

If the path were to be removed entirely:
```
#SBATCH --output=ATACcalcQCS2-%A_%a.log
#SBATCH --output=ATACcalcQCS2-%A_%a.err
```

Then, the script can be ran from anywhere (and won't fail if you don't run from the root of the repository)

[Sorry if this is a repeat comment]